proptest/README.md

```diff
@@ -27,6 +27,12 @@ sees passive maintenance.
 See the [changelog](https://github.com/AltSysrq/proptest/blob/master/proptest/CHANGELOG.md)
 for a full list of substantial historical changes, breaking and otherwise.
 
+### MSRV
+
+The current MSRV of this crate is 1.60.
```

Comment:
we can just put it in the cargo file:
https://doc.rust-lang.org/nightly/cargo/reference/manifest.html#the-rust-version-field
and not worry about needing to update the readme

Reply:
made that change here https://github.com/proptest-rs/proptest/pull/307